src/item_factory.cpp

```diff
@@ -1319,8 +1319,11 @@ void Item_factory::finalize()
             it->second.recipes.push_back( p.first );
         }
     }
+    for( auto &e : m_template_groups ) {
+        auto &isd = e.second;
+        isd->finalize( itype_id::NULL_ID() );
```

Comment:
Did a small test with passing `e.first.str()` to `finalize` and it gave me output like this:

```
03:21:13.768 ERROR : [finalize] entry in itemgroup toxic_waste_mx_radioactive for spawning item cobalt_60 defined charges but can't have any
03:21:13.768 ERROR : [finalize] entry in itemgroup toxic_waste_mx_radioactive for spawning item plutonium defined charges but can't have any
03:21:13.783 ERROR : [finalize] entry in itemgroup zombie_reenactor_drops for spawning item tobacco defined charges but can't have any
03:21:13.783 ERROR : [finalize] entry in itemgroup zombie_reenactor_drops for spawning item hardtack defined charges but can't have any
03:21:13.783 ERROR : [finalize] entry in itemgroup ÃÀ 116 for spawning item cigar defined charges but can't have any
03:21:13.798 ERROR : [finalize] entry in itemgroup ÃÀ 1200 for spawning item 2x4 defined charges but can't have any
03:21:13.798 ERROR : [finalize] entry in itemgroup ÃÀ 1230 for spawning item pipe defined charges but can't have any
03:21:13.798 ERROR : [finalize] entry in itemgroup ÃÀ 1260 for spawning item splinter defined charges but can't have any
```

Doesn't help with the completely anonymous groups, but it's at least something.

Reply:
Thanks, I'd like to implement this but I'm pretty busy irl, and I believe these debugmsg will hardly appear in the future ( sorry to 3rd-party mod maintainers ), so...perhaps this can be kept for future.